src/ioctls/vm.rs

```diff
@@ -812,4 +855,25 @@ mod tests {
         let msi = kvm_msi::default();
         assert!(vm.signal_msi(msi).is_err());
     }
+
+    /// Providing the `kvm_enable_cap` structure filled with default() should
```

Comment:
Can you move the comments before the assert? I don't think that's a common practice in Rust to write the comments before the test function. I haven't seen it at least.